x-pack/solutions/security/plugins/cloud_security_posture/server/routes/graph/utils/esql.utils.ts

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
File name doesn't follow kibana's style guide. Better to keep consistent and change to `snake_casing` ([see reference](https://github.com/elastic/kibana/blob/main/STYLEGUIDE.mdx#filenames))

<img width="435" height="163" alt="Image" src="https://github.com/user-attachments/assets/31716c5e-86c2-4a8e-867b-35904ec2bbd2" />